cmd/controller-manager/app/controller-manager.go

```diff
@@ -107,12 +109,28 @@ func Run(opts *options.Options, stopChan <-chan struct{}) error {
 		panic(err)
 	}
 
-	setOptionsByKubeFedConfig(opts)
-
 	if err := utilfeature.DefaultFeatureGate.SetFromMap(opts.FeatureGates); err != nil {
```

Comment:
This check needs to occur after `setOptionsByKubeFedConfig` is called because it depends on `opts.FeatureGates` being set by the call.

Reply:
Ack